src/main/java/de/tum/in/www1/artemis/repository/JsonAssessmentRepository.java

```diff
@@ -27,10 +33,30 @@ private Path getPath(long exerciseId, long studentId, long modelId, boolean manu
      * @param assessment the assessment as string in a JSON format
      * @return success / failure
      */
+    //TODO merge both writeAssessment Methods?
     public boolean writeAssessment(long exerciseId, long studentId, long modelId, boolean manual, String assessment) {
         return this.write(this.getPath(exerciseId, studentId, modelId, manual), assessment);
     }
 
+    /**
+     * Write an assessment with the following attributes
+     *
+     * @param exerciseId exerciseId
+     * @param studentId studentId
+     * @param modelId modelId
+     * @param modelingAssessment the assessment as list of the assessed elements
+     * @return success / failure
+     */
+    public boolean writeAssessment(long exerciseId, long studentId, long modelId, boolean manual, List<ModelElementAssessment> modelingAssessment) {
+        String modelingAssessmentString = "{\"assessments\":";
+        try {
+            modelingAssessmentString += mapper.writeValueAsString(modelingAssessment) + "}";
+        } catch (JsonProcessingException e) {
+            e.printStackTrace(); //TODO throw Exception
```

Comment:
What about this TODO? Do we need to throw an exception or should we remove the TODO?
Please make sure that the stack trace is printed on the console using the logger

Reply:
since the assessment cant be stored in case of this exception the server shouldnt return a 200 OK. So throwing an Exception seems adequate. I log the error now and throw the InternalServerErrorException so the server returns a 500 in case of an error during save. Since the Json repositories are about to be removed, i ignored other cases in which errors during operations are only logged but do not influence the request outcome.